js/ui/layout.js

```diff
@@ -537,10 +540,30 @@ var LayoutManager = new Lang.Class({
             this._systemBackground.actor.show();
             global.stage.show();
 
-            this._prepareStartupAnimation();
+            this._backgroundLoaded = true;
+
+            this._maybePrepareStartupAnimation();
         }));
     },
 
+    _loadParentalControls: function() {
+        let parentalControlsManager = ParentalControlsManager.getDefault();
+
+        if (!parentalControlsManager.initialized)
+            parentalControlsManager.connect('initialized', this._maybePrepareStartupAnimation.bind(this));
```

Comment:
Maybe this should be disconnected once `_maybePrepareStartupAnimation()` succeeds?